### PR TITLE
Add Swagger API Spec

### PR DESCRIPTION
**Pull request by OR13**
Addresses: https://github.com/decentralized-identity/sidetree/issues/492

### CODE REVIEW

File: docs/swagger/api.yml [hunk 1]
```diff
@@ -0,0 +1,620 @@
...
+# Added by API Auto Mocking Plugin
+servers:
+  - description: SwaggerHub API Auto Mocking
+    url: https://virtserver.swaggerhub.com/sidetree/0.0.1
+info:
+  description: This is an experimental Open API Specification for the Sidetree Protocol.
+  version: "0.0.1"
+  title: Sidetree REST API
+tags:
+  - name: Node
```
Comment (troyronda) [inline docs/swagger/api.yml:11]: @OR13 Where are the DID Method operations going?
https://github.com/decentralized-identity/sidetree/blob/master/docs/protocol.md#sidetree-rest-api

Comment (OR13) [reply docs/swagger/api.yml:11]: I think they are entirely missing from this.

Comment (troyronda) [reply docs/swagger/api.yml:11]: @OR13 awesome. thanks!

Comment (OR13) [reply docs/swagger/api.yml]: I added some endpoints, but IMO we need to define the JWS in the Core Spec, not in the API interface (compact JWS is normative)

**Comment by OR13** [comment]
I'm gonna merge this, because its WIP, and not referenced yet.

**Pull request merged**